CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+$ git remote add upstream git@github.com:symfony/ux.git
+```
+
+### Setting up the development environment
+
+To set up the development environment, you need the following tools:
+
+- [PHP](https://www.php.net/downloads.php) 8.1 or higher
+- [Composer](https://getcomposer.org/download/)
+- [Node.js](https://nodejs.org/en/download/package-manager) 22 or higher
+- [Yarn](https://yarnpkg.com/) 4 or higher
+
+With these tools installed, you can install the project dependencies:
+
+```shell
+$ composer install
+$ yarn install
+```
+
+### Linking Symfony UX packages to your project
+
+If you want to test your code in an existing project that uses Symfony UX packages,
+you can use the `link` utility provided in this Git repository (that you have to clone).
+
+This tool scans the `vendor/` directory of your project, finds Symfony UX packages it uses,
+and replaces them by symbolic links to the ones in the Git repository.
+
+```shell
+$ php link /path/to/your/project
+```
+
+### Working with PHP code
+
+Symfony UX is first and foremost a PHP project, and we follow the [Symfony coding standards](https://symfony.com/doc/current/contributing/code/standards.html)
+and [the Backward Compatibility Promise](https://symfony.com/doc/current/contributing/code/bc.html).
+
+When contributing, please make sure to follow these standards and to write tests for your code,
+runnable with `php vendor/bin/simple-phpunit`.
+
+### Working with assets
+
+Assets are specific to each Symfony UX package:
+  - They are located in the `assets/` directory of each package, and can be either TypeScript or CSS files, respectively compiled through Rollup and PostCSS,
+  - Assets are mentioned in the `package.json` file of each package,
+  - Assets **must be** compiled before committing changes,
+  - Assets **must be** compatible with the [Symfony AssetMapper](https://symfony.com/doc/current/frontend/asset_mapper.html) and [Symfony Webpack Encore](https://symfony.com/doc/current/frontend/encore/index.html).
+
+To help you with assets, we provide the following commands that can be run at the root of the repository or in each package directory:
+  - `yarn run build`: build (compile) assets,
+  - `yarn run test`: run the tests (through Vitest),
+  - `yarn run lint`: lint assets (through Biome.js),
+  - `yarn run format`: format assets (through Biome.js),
+  - `yarn run check-lint`: check assets linting (through Biome.js),
+  - `yarn run check-format`: check assets formatting (through Biome.js)
+
+## Useful Git commands
+
+1. To keep your fork up-to-date with the upstream repository and `2.x` branch, you can run the following commands:
+```shell
+$ git checkout 2.x && \
+  git fetch upstream && \
+  git rebase upstream/2.x && \
+  git push origin 2.x
+```
+
+2. To rebase your branch on top of the `2.x` branch, you can run the following commands:
+```shell
+$ git checkout my-feature-branch && \
+  git rebase upstream/2.x && \
+  git push -u origin my-feature-branch
+```
```

Comment:
Just a note for late: what about addind gh cli usage ? 

```
gh repo sync
```

Reply:
It was one of my first attentions, but... 🤷🏻 

I always use `gco 2.x && gf upstream && grb upstream/2.x && git push` 😅

Reply:
`sfux`    😆